gcp-cloud-run/cloud_run.tf

```diff
@@ -0,0 +1,140 @@
+
+################################
+### Cloud Run (Vault Server) ###
+################################
+
+locals {
+  vault_region = coalesce(var.vault_region, var.default_region)
+}
+
+resource "google_cloud_run_service" "pvault-server" {
+  name     = "${var.deployment_id}-pvault-server"
+  location = local.vault_region
+  provider = google-beta
+
+  metadata {
+    annotations = {
+      "run.googleapis.com/client-name" = "terraform"
+      "run.googleapis.com/ingress"     = "internal"
+    }
+  }
+
+  template {
+    spec {
+      timeout_seconds       = 28
+      container_concurrency = 100
+      containers {
+        image = "${var.image}:${var.vault_version}"
+        env {
+          name  = "PVAULT_TLS_ENABLE"
+          value = "false"
+        }
+        env {
+          name  = "PVAULT_DB_REQUIRE_TLS"
+          value = "false"
+        }
+        env {
+          name  = "PVAULT_DB_HOSTNAME"
+          value = module.postgresql-db.private_ip_address
```

Comment:
I think that you should use `module.postgres-db.instance_self_link` and then you can turn `PVAULT_DB_REQUIRE_TLS` to `true`.